platform/darwin/src/MGLStyleLayer.h.ejs

```diff
@@ -0,0 +1,29 @@
+<%
+  const type = locals.type;
+  const layoutProperties = locals.layoutProperties;
+  const paintProperties = locals.paintProperties;
+-%>
+// This file is generated. Do not edit.
+
+#import <Mapbox/Mapbox.h>
+
+#import "MGLTypes.h"
+
+@interface MGL<%- camelize(type) %>StyleLayer : MGLStyleLayer
+
+<% if (layoutProperties.length) { -%>
+// Layout properties
```

Comment:
Nit: Use `#pragma mark Layout Properties` so jazzy can create a new section for the symbols below.